src/ucm/cuda/cudamem.c

```diff
@@ -452,13 +454,65 @@ static void ucm_cudamem_get_existing_alloc(ucm_event_handler_t *handler)
     }
 }
 
+ucs_status_t ucm_cuda_get_current_device_info(ucs_sys_bus_id_t *bus_id)
+{
+    static ucs_sys_bus_id_t cached_bus_id = {0xffff, 0xff, 0xff, 0xff};
+    CUresult cu_err;
+    CUdevice cuda_device;
+    CUpointer_attribute attribute;
+    int attr_result;
+
+    ucm_trace("ucm_cuda_get_current_device_info");
+
+    if (cached_bus_id.slot == 0xff) {
```

Comment:
remove empty block. just check c`ached_bus_id.slot != 0xff` and return cache hit

Reply:
seems not handled

Reply:
seems this comment is not handled